### PR TITLE
Bounties

### PR DESCRIPTION
**Pull request by articobandurini**
This is a new Bounties page which has following features:
- caching: all issues got directly from github from several repositories, which has limits for unauthicated queries (60 per hour). to address this limitation all results being cached for an hour into localStorage and caching time is shown to the user.
- ordering: all bounties are being parsed which allows for ordering by different parameters. list of parameters can be easily expanded - for now i used only the most obvoius ones.
- opening: bounties can be clicked and quickly opened with markdown parsing to view its content.
- urgent: there is a possibility to get only bounties with the label "urgent" - specially for "urgent tasks" block for the landing page - if this page is okay implementing that block is easy and obvious.

### CODE REVIEW

**Comment by troggy** [comment]
Observed an issue: shows "Loading.." forever on a very first opening. Have to refresh the page to see bounties

File: scripts/bounties.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 	const urgentOnly = false;
 	const useAuth = false;
-	const token = 'c7fe8835417fc0d60229b3983c1a076240e9bca0';
```
Comment (troggy) [inline scripts/bounties.js]: don't forget to invalidate this token

**Comment by sunify** [comment]
Imagine questions people have on this page

1. What is about? People have no idea how our bounties works. How to get payed? How to take a bounty? We have some description on bounty board — at least, we can take it.
2. What I can do? What bounties are free to take?
3. What needs to be reviewed (my personal concern)?

None of these questions are answered

---

Some thoughts on bounties formatting:
1. Do not show bounties without size tag
2. No need for popup, we have this info on github + maybe some discussion
3. Maybe would be better to group bounties by repos
4. Not clear what sizing means. We need some legend for that
5. No need for «Cached until»
6. Useful sort options: size, date (freshness), repo (maybe, see 3.), shares
7. Cards do not seem as a good format for such info. List or table can work better

File: earn.html [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+          <p>
+            Earn with LeapDAO by completing a bounty, taking on a role, or applying for a grant.
+          </p>
+        </header>
+        
+        <h2>Grants</h2>
+        <div class="cols">
+          <div class="col">
+            <p>
+              LeapDAO cares about mass adoption of decentralized future,
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
              Mass adoption is at the core of the LeapDAO purpose.
```

File: earn.html [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+              Leap Network is a step towards it and we are going to support
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
              Scalable and secure decentralized Applications can deliver this vision.
```

File: earn.html [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+              dApps that use our Plasma solution with grants.
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
              Hence we support selected entrepreneurs and teams Plasma with grants.
```

File: earn.html [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+            </p>
+          </div>
+          <div class="col">
+            <p>
+              If you want to build a dApp which leverages high performance of
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
              If you are an entrepreneur or team delivering a use-case
```

File: earn.html [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+              our network and helps to showcase its benefits – we can fund
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
               that requires fast transaction and/or complex computation,
```

File: earn.html [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+              such a project.
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
               reach out to us, and we will support you with mentorship and funding. 
```

File: earn.html [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+              <br/>Talk to us on <a href="https://docs.google.com/forms/d/e/1FAIpQLSd8_wDGDAi__HvfYEWNK_bvJzIkxwHHRVL6AFEfJewBd2Vn9A/viewform">Slack</a>.
+            </p>
+          </div>
+        </div>
+        
+        <h2>Roles</h2>
+        <div class="cols">
+          <div class="col">
+            <p>
+              LeapDAO is an open community which runs on
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
              LeapDAO is an open community operating on 
```

File: earn.html [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+          </div>
+        </div>
+        
+        <h2>Roles</h2>
+        <div class="cols">
+          <div class="col">
+            <p>
+              LeapDAO is an open community which runs on 
+              <a href="https://www.holacracy.org/" target="_blank">Holacracy</a> 
+              – non-hierarchical governance methodology.
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
              – a non-hierarchical governance methodology.
```

File: earn.html [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+              There are no job positions, but roles, 
+              and every member holds one or more of them.
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
              and every partner holds one or more of them.
```

File: earn.html [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+            <p>Loading...</p>
+        </div>
+        
+        <h2>Bounties</h2>
+        <div class="cols">
+          <div class="col">
+            <p>
+              At LeapDAO we use bounties to reward task completion.
+              Take an unassigned bounty from the list, work on it, get reviewed
+               and earn money! We pay in DAI stablecoin, which rate is equal to US dollar.
```
Comment (johannbarbie) [inline earn.html]:
```suggestion
               and earn money! We pay in DAI, a stablecoin pegged to the US dollar.
```

**Pull request merged**